core/inspector.rs

```diff
@@ -547,11 +563,8 @@ impl InspectorSession {
   pub fn break_on_next_statement(&mut self) {
     let reason = v8::inspector::StringView::from(&b"debugCommand"[..]);
     let detail = v8::inspector::StringView::empty();
-    self
-      .v8_session
-      .borrow_mut()
-      .as_mut()
-      .schedule_pause_on_next_statement(reason, detail);
+
+    (*self.v8_session).schedule_pause_on_next_statement(reason, detail);
```

Comment:
Not quite right either, considering that v8_session derefs to a &mut which may be aliased. Rusty_v8 should probably have a `UniquePtr::as_mut_ptr()` method but it doesn't.

Reply:
How do we handle that in the meantime then?

Reply:
> How do we handle that in the meantime then?

There doesn't seem to be a good solution. 
Maybe add `// TODO: use raw `*mut V8InspectorSession` pointer, as this reference may become aliased.`